htlcswitch/interfaces.go

```diff
@@ -160,6 +161,16 @@ type ChannelUpdateHandler interface {
 	// will only ever be called once. If no CommitSig is owed in the
 	// argument's LinkDirection, then we will call this hook immediately.
 	OnCommitOnce(LinkDirection, func())
+
+	// InitStfu allows us to initiate quiescence on this link. It returns
+	// a receive only channel that will block until quiescence has been
+	// achieved, or definitively fails.
+	//
+	// This operation has been added to allow channels to be quiesced via
+	// RPC. It may be removed or reworked in the future as RPC initiated
+	// quiescence is a holdover until we have downstream protocols that use
+	// it.
+	InitStfu() <-chan fn.Option[bool]
```

Comment:
If the link is shutting down, do we close the channel that we return from this method? Assuming it's a global instance, then that would allow a caller to break out when the link disconnects.

Reply:
Ok looks like yeah below, we should add that to the `godoc`. To ensure that callers aways do the two value read. 

What does the option mean in this case? What's expeted behavior if the caller gets `None` here? Does that mean that we can't STFU, or that one is already on going, or other?

Reply:
Checked the next commit, and looks like maybe `fn.Result[bool]` is better here? Then we can also return a precise error re _why_ we failed to `stfu`.